src/scripts/components/main/globe/globe.tsx

```diff
@@ -76,7 +81,9 @@ const Globe: FunctionComponent<Props> = props => {
 
   useProjectionSwitch(globe, projectionState.projection);
   useMultiGlobeSynchronization(globe, props);
-  // eslint-disable-next-line no-warning-comments
+
+  useLayerLoadingStateUpdater(globe, props.onLayerLoadingStateChange);
+
```

Comment:
Could you pls re-add `// eslint-disable-next-line no-warning-comments` or open an issue for this? The lint-and-test run is failing. Also deployment would be blocked :)